tables/iRO/official/items.txt

```diff
@@ -8556,3 +8555,4 @@
 20763#Costume_Amistr_Bag#
 31052#Costume_Alchemist_Mask#
 23164#Granpa_Speed_potion#
+20164#Costume:_Duneyrr_Helm#
```

Comment:
If they're removing and renaming an item like this, does that mean anyone who had that book now has this new item? xD